.docker-compose.datagov-theme.yaml

```diff
@@ -25,9 +27,8 @@ services:
           install_standard_ckan_extension_github -r ckan/ckanext-harvest &&\
           install_standard_ckan_extension_github -r datopian/ckanext-logstash &&\
           install_standard_ckan_extension_github -r ckan/ckanext-dcat ckanext-dcat &&\
-          install_standard_ckan_extension_github -r GSA/ckanext-geodatagov &&\
-          install_standard_ckan_extension_github -r akariv/ckanext-datajson -b datagov &&\
-          install_standard_ckan_extension_github -r akariv/USMetadata -e ckanext-usmetadata &&\
```

Comment:
We still need the USMetadata extension in non-harvester datagov instances.

Reply:
I know, I proposed to create a new yaml file for the **inventory**.
This yaml file is intented to work as **catalog**

Reply:
What I mean is that we need this extension _installed_ - we won't _enable_ it in the harvester kind.

Reply:
I added this ext again.